src/themes/elbil24/index.js

```diff
@@ -0,0 +1,23 @@
+import { cssReset } from '../../utils/css-reset';
+
+import colors from './colors';
+
+const variables = {
+	mainFont: "'Open Sans', helvetica, arial, sans-serif",
+	headingsFont: "'Open Sans', helvetica, arial, sans-serif",
+};
+
+const global = `
+	@import url('https://fonts.googleapis.com/css?family=Open+Sans:300,400,700,800');
```

Comment:
Do you use all these weights? uiWeight, uiWeightBold and uiWeightSemibold is defined in theme.variables - but that's only 3 different weights. Here there are four.